src/auth/redux.js

```diff
@@ -20,6 +20,7 @@ export const AuthActionType = {
   UpdatePassword: 'civicApp/auth/updatePassword',
   UpdateDuplicatePassword: 'civicApp/auth/updateDuplicatePassword',
   ShowErrors: 'civicApp/auth/showErrors',
+  UserRegistered: 'civicApp/auth/userRegistered',
```

Comment:
I think all the registration stuff should go in the user reducer, rather than the auth one.  That way we can save it in firebase in our datastore, rather than having to re-ask them every time (I don't think you can save extra data onto the auth store objects)

Reply:
I think adding this to the user reducer would be good if that makes it easier to store with the other user properties. I don't think we want to always show this screen when users open the app if they haven't registered yet. I think it would be a bad user experience, especially if you're previously registered to vote (outside of civic), to always be presented with this screen. Users don't have to register to vote in order to use our app. But they do have to take the survey and find their district

Reply:
that is a good point - maybe we would show an alert occasionally or something, but that is true that it would be really annoying as a user to be taken to that screen every time

Reply:
Just to clarify, the flow per the mocks is that we show the voter registration screen when the user clicks a top-of-screen alert prompting them to register to vote on the elections page (later, the home page). Definitely agree that we don't want to direct the user here all the time, or even during their initial run-through of the survey. :)